gbfs.md

```diff
@@ -303,7 +305,56 @@ Field Name | REQUIRED | Type | Defines
   }
 }
 ```
+### manifest.json
+An index of `gbfs.json` URLs for each GBFS data set produced by a publisher. A single instance of this file should be published at a single stable URL, for example: `https://example.com/gbfs/manifest.json`
 
+The following fields are all attributes within the main `data` object for this feed.
+
+Field Name | REQUIRED | Type | Defines
+---|---|---|---
+`datasets` | Yes | Array | An array containing
```

Comment:
Wondering if `feeds` would be more appropriate. IIRC it's used a lot in the spec. 

`versions` also refers to feeds to 

> the available versions of a feed

Reply:
I went with `datasets` because there's been disagreement on what constitutes a "feed". Is a feed a single endpoint or a complete collection of endpoints?  If you look at `gbfs.json#feeds` it's defined as an array of individual endpoints which together represent a single set of data.